Tests are failing

```
Failure in test test_install_proposed_upgrades (ftw.upgrade.tests.test_command_install.TestInstallCommand)
Traceback (most recent call last):
  File "/Users/mbaechtold/.buildout/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 340, in run
    testMethod()
  File "/Users/mbaechtold/Developer/Projects/ftw.upgrade/ftw/upgrade/tests/test_command_install.py", line 33, in test_install_proposed_upgrades
    exitcode, output = self.upgrade_script('install -s plone --proposed')
  File "/Users/mbaechtold/Developer/Projects/ftw.upgrade/ftw/upgrade/tests/base.py", line 135, in upgrade_script
    command, assert_exitcode=assert_exitcode)
  File "/Users/mbaechtold/.buildout/eggs/ftw.testing-1.11.0-py2.7.egg/ftw/testing/layer.py", line 179, in execute_script
    exitcode, command, output))
AssertionError: Expected exit code 0, got 1 for "/private/var/folders/74/1sl4n31d4zs5qcfh3dfxlty00000gp/T/tmp1nWSE6buildoutSetUp/_TEST_/sample-buildout/bin/upgrade install -s plone --proposed".
Output:
Traceback (most recent call last):
  File "/private/var/folders/74/1sl4n31d4zs5qcfh3dfxlty00000gp/T/tmp1nWSE6buildoutSetUp/_TEST_/sample-buildout/bin/upgrade", line 267, in <module>
    sys.exit(ftw.upgrade.command.main())
  File "/Users/mbaechtold/Developer/Projects/ftw.upgrade/ftw/upgrade/command/__init__.py", line 197, in main
    UpgradeCommand()()
  File "/Users/mbaechtold/Developer/Projects/ftw.upgrade/ftw/upgrade/command/__init__.py", line 170, in __call__
    args.func(args)
  File "/Users/mbaechtold/Developer/Projects/ftw.upgrade/ftw/upgrade/command/jsonapi.py", line 122, in func_wrapper
    return func(args, requestor)
  File "/Users/mbaechtold/Developer/Projects/ftw.upgrade/ftw/upgrade/command/jsonapi.py", line 131, in func_wrapper
    return func(*args, **kwargs)
  File "/Users/mbaechtold/Developer/Projects/ftw.upgrade/ftw/upgrade/command/install.py", line 105, in install_command
    for line in response.iter_lines(chunk_size=30, decode_unicode=True):
  File "/Users/mbaechtold/.buildout/eggs/requests-2.11.0-py2.7.egg/requests/models.py", line 720, in iter_lines
    for chunk in self.iter_content(chunk_size=chunk_size, decode_unicode=decode_unicode):
  File "/Users/mbaechtold/.buildout/eggs/requests-2.11.0-py2.7.egg/requests/utils.py", line 374, in stream_decode_response_unicode
    for chunk in iterator:
  File "/Users/mbaechtold/.buildout/eggs/requests-2.11.0-py2.7.egg/requests/models.py", line 676, in generate
    for chunk in self.raw.stream(chunk_size, decode_content=True):
  File "/Users/mbaechtold/.buildout/eggs/requests-2.11.0-py2.7.egg/requests/packages/urllib3/response.py", line 353, in stream
    for line in self.read_chunked(amt, decode_content=decode_content):
  File "/Users/mbaechtold/.buildout/eggs/requests-2.11.0-py2.7.egg/requests/packages/urllib3/response.py", line 521, in read_chunked
    line = self._fp.fp.readline()
AttributeError: 'NoneType' object has no attribute 'readline'
```

--- a/docs/HISTORY.txt
+++ b/docs/HISTORY.txt
@@ -5,7 +5,7 @@ Changelog
 1.19.1 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Fix NoneType AttributeError with newest requests module. [jone]
 
 
 1.19.0 (2016-04-11)

--- a/ftw/upgrade/command/install.py
+++ b/ftw/upgrade/command/install.py
@@ -102,7 +102,10 @@ def install_command(args, requestor):
 
     with closing(requestor.POST(action, params=params,
                                 stream=True)) as response:
-        for line in response.iter_lines(chunk_size=30, decode_unicode=True):
+        for line in response.iter_lines(chunk_size=30):
+            if isinstance(line, str):
+                line = line.decode('utf-8')
+
             print line
 
     if line.strip() != 'Result: SUCCESS':

--- a/ftw/upgrade/command/plone_upgrade.py
+++ b/ftw/upgrade/command/plone_upgrade.py
@@ -44,7 +44,10 @@ def plone_upgrade_command(args, requestor):
 
     with closing(requestor.POST(action, params=params,
                                 stream=True)) as response:
-        for line in response.iter_lines(chunk_size=30, decode_unicode=True):
+        for line in response.iter_lines(chunk_size=30):
+            if isinstance(line, str):
+                line = line.decode('utf-8')
+
             print line
 
     line = line.strip()

